cabby/rvs/templates.py

```diff
@@ -301,5 +309,21 @@ def add_features_to_template(template: Text, entity: geo_item.GeoEntity) -> Text
   template = template.replace(' a o', ' an o')
   template = template.replace('_', ' ')
 
-  return template
+  entities_span_list = {}
```

Comment:
This is a dict, so _list seems like a bad name. How about just calling it entity_spans?

Reply:
Changed to entities_span_dict